cmd/ctr/commands/images/images.go

```diff
@@ -115,29 +124,47 @@ var listCommand = cli.Command{
 				platformColumn = strings.Join(ps, ",")
 			}
 
-			labels := "-"
-			if len(image.Labels) > 0 {
-				var pairs []string
-				for k, v := range image.Labels {
-					pairs = append(pairs, fmt.Sprintf("%v=%v", k, v))
-				}
-				sort.Strings(pairs)
-				labels = strings.Join(pairs, ",")
+			iPrint := &imagePrint{
+				imageName:  image.Name,
+				digest:     image.Target.Digest,
+				size:       progress.Bytes(size),
+				platform:   platformColumn,
+				createTime: image.CreatedAt.In(local).Format("2006-01-02 15:04:05"),
+				createdAt:  image.CreatedAt.UnixNano(),
 			}
+			imagePrintArray = append(imagePrintArray, iPrint)
+		}
 
-			fmt.Fprintf(tw, "%v\t%v\t%v\t%v\t%v\t%s\t\n",
-				image.Name,
-				image.Target.MediaType,
-				image.Target.Digest,
-				progress.Bytes(size),
-				platformColumn,
-				labels)
+		sort.Sort(imagePrintArray)
```

Comment:
Why sort?

Reply:
Sorting images according to createdTime can obtain a stable display order, and the result is almost similar to the display habit of Docker